src/Tribe/Query.php

```diff
@@ -27,6 +27,17 @@ class Tribe__Events__Query {
 	 * @param WP_Query $query
 	 */
 	public static function parse_query( $query ) {
+
+		// Bail if we're not dealing with any event or event related queries.
+		if ( 
+			! tribe_is_event_query()
+			|| ! tribe_is_event_organizer()
+			|| ! tribe_is_event_venue()
+			|| ! ( $post instanceof WP_Post && has_shortcode( $post->post_content, 'tribe_events' ) )
```

Comment:
$post is not defined - is this the global `$post`? Then we need to add it prior to this to be sure we know what we're dealing with. i.e. `global $post` before the if.